--- toycars-0.3.5/src/RobotHotPotatoPlayer.cpp~	2007-12-18 14:13:22.000000000 +0100
+++ toycars-0.3.5/src/RobotHotPotatoPlayer.cpp	2007-12-25 23:53:02.000000000 +0100
@@ -16,6 +16,7 @@
 
 #include "RobotHotPotatoPlayer.h"
 #include "HotPotatoGame.h"
+#include <assert.h>
 
 RobotHotPotatoPlayer::RobotHotPotatoPlayer(ToyCar& inCar, list<Tuple> *path, int inPlayerNumber, FMOD::System* inFMOD_System)
 	: RobotPlayer(inCar, path, inPlayerNumber, inFMOD_System), backupDist(-1.0)
