--- toycars-0.3.7/src/Player.cpp~	2008-06-29 17:53:51.000000000 +0200
+++ toycars-0.3.7/src/Player.cpp	2008-11-25 14:28:55.000000000 +0100
@@ -14,6 +14,8 @@
  *  Created by Ruben Henner Zilibowitz on 2/01/05.
  */
 
+#include <stdio.h>
+
 #include "Player.h"
 #include "CarGame.h"
 
--- toycars-0.3.7/src/Startline.cpp~	2007-12-25 15:21:33.000000000 +0100
+++ toycars-0.3.7/src/Startline.cpp	2008-11-25 14:32:54.000000000 +0100
@@ -14,6 +14,8 @@
  *  Created by Ruben Henner Zilibowitz on 19/04/05.
  */
 
+#include <stdio.h>
+
 #include "Startline.h"
 #include "ScException.h"
 #include <algorithm>
--- toycars-0.3.7/src/KnockOutGame.cpp~	2008-06-09 16:14:46.000000000 +0200
+++ toycars-0.3.7/src/KnockOutGame.cpp	2008-11-25 14:36:35.000000000 +0100
@@ -7,6 +7,8 @@
  *
  */
 
+#include <stdio.h>
+
 #include "KnockOutGame.h"
 #include "GameConstants.h"
 #include "ScException.h"
--- toycars-0.3.7/src/RobotHotPotatoPlayer.cpp~	2008-06-09 09:43:47.000000000 +0200
+++ toycars-0.3.7/src/RobotHotPotatoPlayer.cpp	2008-11-25 14:39:59.000000000 +0100
@@ -14,6 +14,8 @@
  *  Created by Ruben Henner Zilibowitz on 31/10/07.
  */
 
+#include <stdio.h>
+
 #include "RobotHotPotatoPlayer.h"
 #include "HotPotatoGame.h"
 #include <cassert>
